Makefile

```diff
@@ -77,8 +77,8 @@ endif
 
 ifeq "$(WITH_SS11)" "1"
 	ifeq ($(HPC_LIBS_DIR),)
-           LIBFAB_SO=$(shell find $(CRAY_LIBFABRIC_DIR) -name libfabric\*so)
-           LIBCXI_SO=$(shell find $(CRAY_LIBCXI_DIR) -name libcxi\*so)
+           LIBFAB_SO=$(shell find $(CRAY_LIBFABRIC_DIR) -name libfabric\*so.\*)
```

Comment:
I Will there ever be more than one libfabric? Like
libfabric.so.1
libfabric.so.1.0
libfabric.so.1.0.0
?

Reply:
Likely. I think your suggestion is fine though. All we're trying to do here is find a directory containing them automatically but user's can provide the dir if there are multiple ones. Though, with this PR the point is for us to pull the GitHub source and build it that way without user input or system libs. I should make WITH_SS11 0 by default now to prevent us pulling it from the host if we're building on an EX.